### ISSUE TITLE
aria 508c accessibility p-multiselect fails to articulate unchecked for select ALL

### ISSUE THREAD

**Issue by kwc-scopeinfotech** [primefaces/primeng#10485]
This is a bug for release 11.4.2

**Current behavior**
aria 508c accessibility p-multiselect fails to articulate unchecked for select ALL

ACTION1 
Upon select-all checkbox in the 

- p-multiselect-header screen reader accurately speaks the amount of items selected (checked)

ACTION2
Upon de-select-all in the 

- p-multiselect-header screen reader repeats speech of ACTION1 when it should say 0 items checked

**Expected behavior**

**Minimal reproduction of the problem with instructions**
render a p-multiselect as follows

```
        <div #container [ngClass]="{'p-multiselect p-component':true,
            'p-multiselect-open':overlayVisible,
            'p-multiselect-chip': display === 'chip',
            'p-focus':focus,
            'p-disabled': disabled}" [ngStyle]="style" [class]="styleClass"
            (click)="onMouseclick($event,in)">
            <div
                itemLabel="Checkbox"
                class="p-hidden-accessible">
                <input #in type="text" readonly="readonly" [attr.id]="inputId" [attr.name]="name" (focus)="onInputFocus($event)" (blur)="onInputBlur($event)"
                       [disabled]="disabled" [attr.tabindex]="tabindex" (keydown)="onKeydown($event)" aria-haspopup="listbox" [attr.aria-expanded]="overlayVisible"
                       [attr.aria-labelledby]="ariaLabelledBy"
                       itemLabel="Checkbox"
                       [attr.aria-roledescription]="'Select all items'" aria-live="polite"
                       role="checkbox" tabindex="0"
                       aria-atomic="true" aria-label="" aria-describedby="checkboxDrama" >
                <span [hidden]="true" aria-label="checkbox"  value="Checkbox" for="Checkbox"></span>
            </div>
```

**What is the motivation / use case for changing the behavior?**
508c govt compliance

Impact
Blind user will think all items are checked and selected when they are not

**Please tell us about your environment:**
MACOS + google screen reader

* **Angular version:** 11

* **PrimeNG version:** 11.4.2

* **Browser:**  CHROME
 
```
kennethcolassi@Scope-2021-010 primeng % ng --version
Your global Angular CLI version (12.1.3) is greater than your local version (11.0.7). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 11.0.7
Node: 14.17.3
OS: darwin x64

Angular: 11.0.9
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.1100.7
@angular-devkit/build-angular      0.1100.7
@angular-devkit/build-ng-packagr   0.1002.0
@angular-devkit/core               11.0.7
@angular-devkit/schematics         11.0.7
@angular/cdk                       11.0.4
@angular/cli                       11.0.7
@schematics/angular                11.0.7
@schematics/update                 0.1100.7
ng-packagr                         11.0.3
rxjs                               6.6.7
typescript                         4.0.8


kennethcolassi@Scope-2021-010 primeng % node --version
v14.17.3
```

**Comment by yigitfindikli** [comment]
We planned full accessibility support for the next versions of PrimeNG. Thank you for your report. PrimeNG will be fully accessible soon.

**Comment by kwc-scopeinfotech** [comment]
Hello yigitfindikli

can you give us a status on when its safe to upgrade to angular 12 and primeng latest?

We want to ditch our fork and go with primeng if it is in fact 508c compliant

you stated SOON... we want this asap
we appreciate all your diligent work

Please advise... thank you
